### PR TITLE
FEAT : added mockbee and rendering products

### PR DESCRIPTION
**Pull request by zoyatanweer**
- added mockbee to the project
- rendered products from backend

Files to check:
- components/products/Products.jsx
- Context/data/Data-Context.jsx

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...lcon-store-react\\falcon-store-react\\src\\components\\products\\FilterSection.js",[],"C:\\Users\\hp\\Documents\\VScode\\neog'22\\falcon-store-react\\falcon-store-react\\src\\components\\index.js",[],"C:\\Users\\hp\\Documents\\VScode\\neog'22\\falcon-store-react\\falcon-store-react\\src\\components\\wishlist\\WishlistData.js",[],"C:\\Users\\hp\\Documents\\VScode\\neog'22\\falcon-store-react\\falcon-store-react\\src\\components\\cart\\CartTotal.jsx",[],"C:\\Users\\hp\\Documents\\VScode\\neog'22\\falcon-store-react\\falcon-store-react\\src\\Assets\\Svg\\allsvg.jsx",[],"C:\\Users\\hp\\Documents\\VScode\\neog'22\\falcon-store-react\\falcon-store-react\\src\\components\\header\\NavLinkItems.jsx",[],"C:\\Users\\hp\\Documents\\VScode\\neog'22\\falcon-store-react\\falcon-store-react\\src\\components\\header\\NavLinkUser.jsx",[],"C:\\Users\\hp\\Documents\\VScode\\neog'22\\falcon-store-react\\falcon-store-react\\src\\components\\homepage\\FeaturedItems.jsx",[],"C:\\Users\\hp\\Documents\\VScode\\neog'22\\falcon-store-react\\falcon-store-react\\src\\Assets\\index.js",[],"C:\\Users\\hp\\Documents\\VScode\\neog'22\\falcon-store-react\\falcon-store-react\\src\\Context\\data\\Data-Context.jsx",[]]
```
Comment (dev-prabhat) [inline .eslintcache:1]: I think this file can be removed

File: src/components/products/Products.jsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import "./Products.css";
-import { data } from "../../data/data";
+// import { Products } from "../../data/data";
```
Comment (dev-prabhat) [inline src/components/products/Products.jsx:3]: removed this commented code

File: src/Context/data/Data-Context.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const getData = async () => {
+    try {
+      const response = await axios.get("/api/products");
+      const responseData = [...response.data.products];
+      setData((previous) => (previous = responseData));
```
Comment (dev-prabhat) [inline src/Context/data/Data-Context.jsx:17]: if this line works fine you can merge

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
some changes required but code is file and clean

**Review by developer-anand — APPROVED** [review]
Nice Clean Code, good to go with changes.

**Review by tonymike99 — APPROVED** [review]
Hi Zoya :)
Everything looks good. 

Suggestion: add the .eslintcache to .gitignore

**Comment by zoyatanweer** [comment]
Thankyou guys for your reviews.

**Pull request merged**